aim/db/models.py

```diff
@@ -666,3 +674,221 @@ class Topology(model_base.Base, model_base.AttributeMixin,
     __tablename__ = 'aim_topologies'
     __table_args__ = (model_base.uniq_column(__tablename__, 'name') +
                       model_base.to_tuple(model_base.Base.__table_args__))
+
+
+class VMMController(model_base.Base, model_base.HasDisplayName,
+                    model_base.HasAimId, model_base.AttributeMixin,
+                    model_base.IsMonitored, model_base.HasName):
+    """DB model for VMM Controller."""
+    __tablename__ = 'aim_vmm_controllers'
+    __table_args__ = (
+        model_base.uniq_column(__tablename__, 'domain_type', 'domain_name',
+                               'name') +
+        (sa.ForeignKeyConstraint(
```

Comment:
I disagree about not adding new ones. It is preferable to maintain consistency with existing code instead of diverging for some cases. If and when we decide to remove FK constraints, it should be trivial to remove them from a few more tables.